posthog/models/tagged_item.py

```diff
@@ -61,6 +61,9 @@ class TaggedItem(UUIDModel):
     action: models.ForeignKey = models.ForeignKey(
         "Action", on_delete=models.CASCADE, null=True, blank=True, related_name="tagged_items"
     )
+    feature_flag: models.ForeignKey = models.ForeignKey(
```

Comment:
Should add a test of feature_flag in `posthog/models/test/test_tagged_item_model.py`